src/components/DefineDissolution/CustodianOfRecords.vue

```diff
@@ -350,6 +345,29 @@ export default class CustodianOfRecords extends Mixins(CommonMixin, EntityFilter
     return this.getCustodian?.inheritMailingAddress
   }
 
+  /** Keep local custodian addresses in sync with base address common component. */
+  private syncAddresses (baseAddress: AddressIF, newAddress: AddressIF): void {
+    Object.assign(baseAddress, newAddress)
```

Comment:
I'm not sure that this does anything. I think you have to assign the result to a variable.

Reply:
This was just reorganized in the code but i can look into this.

Reply:
I don't believe we need to assign it to a new variable. 
Obviously it works as is, but the reason it does so is..
Object.assign takes the source obj and assigns the values to the target obj. keyword being assigns i think. 
The method does return the target, your definitely right there, but since our target is the local model we are updating it, without doing anything with the methods return.

Reply:
OK